handler/core/src/test/java/io/knotx/fragments/handler/action/CircuitBreakerActionTest.java

```diff
@@ -49,20 +59,35 @@
   private static final int TIMEOUT_IN_MS = 500;
 
   @Test
-  @DisplayName("Expect operation ends when doAction ends on time.")
+  @DisplayName("Expect operation ends when doAction ends on time. Action logs from doAction present.")
```

Comment:
Can we split this test to:
- `Expect operation ends when doAction ends on time.`
- `Expect action logs from doAction when doAction ends on time.`
Then when something fails you can read the label and you understand what really happen.

Reply:
done